sdk/go/pulumi/resource.go

```diff
@@ -159,6 +175,10 @@ type resourceOptions struct {
 	Aliases []Alias
 	// AdditionalSecretOutputs is an optional list of output properties to mark as secret.
 	AdditionalSecretOutputs []string
+	// Transformations is an optional list of transformations to apply to this resource during construction.
+	// The transformations are applied in order, and are applied prior to transformation and to parents
+	// walking from the resource up to the stack.
```

Comment:
This comment is a little awkward - is it the same as in other languages (might be my poor writing that’s getting copied in here!).

Reply:
Yup, that comment is taken from the python implementation: https://github.com/pulumi/pulumi/blob/master/sdk/python/lib/pulumi/resource.py#L355-#L359